packages/runtime/test/config.test.js

```diff
@@ -209,3 +209,14 @@ test('parseInspectorOptions()', async (t) => {
     assert.strictEqual(cm.current.inspectorOptions.port, 65535)
   })
 })
+
+test('same schemaOptions as platformatic service', async () => {
+  assert.deepStrictEqual(platformaticRuntime.schemaOptions, platformaticService.schemaOptions)
+})
+
+test('correctly loads the hotReload value from a string', async () => {
+  const configFile = join(fixturesDir, 'configs', 'monorepo-hotreload-env.json')
+  process.env.PLT_HOT_RELOAD = 'true'
+  const loaded = await loadConfig({}, ['-c', configFile], platformaticRuntime)
+  assert.strictEqual(loaded.configManager.current.hotReload, true)
```

Comment:
It would probably be a good idea to test that `'false'` also evaluates to `false`.